app/impl/IdleWrapper.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {XH, HoistComponent, elemFactory} from '@xh/hoist/core';
+
+@HoistComponent()
```

Comment:
Sorry -- you made those last changes so quickly, I hadn't finished my comments. Don't think we need this wrapper component.  This null check can be just made inline in the main renderer.